docs/reference/ambassador-with-aws.md

```diff
@@ -74,7 +79,13 @@ config:
     redirect_cleartext_from: 8080
 ```
 
-If you are running the load balancer in L7 mode, then you will want to redirect all the incoming HTTP requests with `X-FORWARDED-PROTO: http` header to HTTPS. Here is an example Ambassador configuration for this scenario:
+This configuration makes Ambassador start a new listener on 8080 which redirects all cleartext HTTP traffic to HTTPS.
+
+Note: All HTTP requests are redirected to HTTPS, _as is_. For instance, an incoming request to `http://example.com:8888` will be redirected to `https://example.com:8888`. But since our HTTPS listener is on another port (say `9999`), this will fail. So, make sure that your load balancer is configured to listen on standard ports, 80 and 443. Non-standard ports are not supported, [yet!](https://github.com/datawire/ambassador/issues/702)
+
+#### L7 Load Balancer
+
+If you are running the load balancer in L7 mode, then you will want to redirect all the incoming HTTP requests without `X-FORWARDED-PROTO: https` header to HTTPS. Here is an example Ambassador configuration for this scenario:
```

Comment:
without _the_ `X-FORWARDED-PROTO` header (missing "the")

Reply:
fixed